src/layouts/MainLayout.vue

```diff
@@ -1,21 +1,22 @@
 <template>
-  <q-layout view="lHh Lpr lFf">
-    <q-header elevated>
-      <q-toolbar>
+  <q-layout>
+    <q-header :class="headerColor()">
+      <q-toolbar
+        v-if="
+          $route.path == '/learn' || $route.path == `/learn/${$route.params.id}`
```

Comment:
rephrase this as
```suggestion
          ['/learn', `/learn/${$route.params.id}`].find((p) => (p == $route.path))
```
this way you can add more possible path in the future.